task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -1,13 +1,13 @@
----
 apiVersion: tekton.dev/v1
 kind: Task
 metadata:
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: image-build, appstudio, hacbs
+  creationTimestamp: null
```

Comment:
NIT: It would be nice to omit this.

Reply:
Yeah I was a bit puzzled why it is there, but I didn't really want to address it, I guess it might be missing `omitempty`? I'll look into it...

Reply:
The issue with timestamps is that they're using [zero value](https://pkg.go.dev/time#Time.IsZero) of `time.Time`, and the custom Marshaller implementation converts that value to `null`, there is no option to omit `null` values in golang's json. New version of tkn-fmt handles this